docs/styleguide.md

```diff
@@ -2,9 +2,9 @@
 
 From the GitHub Style Guide:
 
-> Words are an important part of how software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
+> Words are an important part of how the software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
 >
->**The way we write is just as important as the way we build. Consider these things when writing copy.**
+>**The way we write is just as important as the way we build. Consider these things when writing a copy.**
```

Comment:
"Copy" here doesn't imply a "duplicate", but a well-known short form for "copywrite" or "copywriting". Hence, the current form of "writing copy" or "copy writing" is correct :)